ModuleOneFirstTaskConsole/Program.cs

```diff
@@ -17,7 +18,9 @@ public class Program
         /// <param name="args">Console attributes.</param>
         public static void Main(string[] args)
         {
-            ConsolePrinter.PrintGreating();
+            var greatingBuilder = new GreatingBuilder();
+            Console.WriteLine(greatingBuilder.Build());
+            Console.ReadKey();
```

Comment:
A `ReadKey` without an informational message can confuse users of your console application.